subjects/atm-management-system/audit/README.md

```diff
@@ -6,73 +6,73 @@
 
 ##### Open the application and re-register the user `"Alice"`.
 
-###### Did it display an error message stating that this user already exist?
+###### Did the application display an error message stating that this user already exists?
 
 ##### Open the file `"./data/users.txt"`.
 
-###### Are all the user's names unique? (ex: no repetition on the name Alice)
+###### Are all the user names unique? (ex: no repetition on the name Alice)
 
 ##### Try and login as `"Alice"`.
 
 ###### Was Alice able to enter the main menu?
 
-##### Try to create two accounts using the user Alice, then select the option `"Update information of account"` and select an account number that does not exist for Alice.
+##### Try to create two accounts with the user Alice, then select the option `"Update account information"` and select an account number that does not exist for Alice.
 
-###### Did the application displayed some kind of error message stating that this account does not exist?
+###### Did the application display some kind of error message stating that this account does not exist?
 
-##### Resorting to the user Alice, try and select the option `"Update information of account"` and select one of the accounts you created for Alice.
+##### Referring to the user Alice, select the option `"Update account information"` and select one of the accounts you created for Alice.
```

Comment:
The "referring" is not the best option i think "Using" is better
Because resorting and using is basically the same thing but  referring is different